src/data/cards/amex.ts

```diff
@@ -494,6 +494,12 @@ export const AMERICAN_EXPRESS_CARDS: z.input<typeof schemas.CreditCard>[] = [
         days: 90,
         credits: [],
       },
+      {
+        amount: [{ amount: 90_000 }],
+        spend: 6000,
+        days: 180,
+        credits: [],
+      },
```

Comment:
Can't reproduce this one, believe this offer ended a few weeks ago

(granted, I hadn't gotten around to this, so it's not your fault)